BcContainerHelper.psd1

```diff
@@ -266,10 +266,10 @@ PrivateData = @{
         # Tags = @()
 
         # A URL to the license for this module.
-        # LicenseUri = ''

```

Comment:
please remove the changes to the .psd1 files (the other ones have become binary)
![image](https://github.com/microsoft/navcontainerhelper/assets/10775043/825e01da-d9cf-4d66-9df4-0d87b172e148)